packages/blocks/src/root-block/widgets/element-toolbar/change-note-button.ts

```diff
@@ -226,7 +226,7 @@ export class EdgelessChangeNoteButton extends WithDisposable(LitElement) {
   @query('.display-mode-button-group')
   private _displayModeButtonGroup!: HTMLDivElement;
   @query('note-display-mode-panel')
-  private _displayModePanel!: HTMLDivElement;
+  private _displayModePanel!: HTMLDivElement | null;
```

Comment:
```ts
private _displayModePanel: HTMLDivElement | undefined;
```
Remove that non-null assertion operator (postfix !)

Reply:
I think the `@query` always returns an `Element` or `null` according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector#return_value). If I use `HTMLDivElement | null` and remove the postfix !, ESLint will raise a "no initializer" error (ts 2564)